build.gradle

```diff
@@ -119,10 +129,10 @@ def isNonStable = { String version ->
     def stableKeyword = ['RELEASE', 'FINAL', 'GA'].any { it -> version.toUpperCase().contains(it) }
     def regex = /^[0-9,.v-]+(-r)?$/
     return !stableKeyword && !(version ==~ regex)
-}
+} as Object
```

Comment:
is this needed?

Reply:
my IntelliJ doesn't seem to care though 🤔

Reply:
Go figure. IntelliJ sometimes acts up a bit with Gradle. Or maybe you have the specific inspection turned off. Anyway, want me to revert it?

Reply:
yes, thanks!

Reply:
Done